### PR TITLE
feature/add-articles-style

### PR DESCRIPTION
**Pull request by KennethanCeyer**
## :rocket: Feature

- Update style for articles/
- Add description section
- Adjust colors

## 🖼️ Screenshot

image

### CODE REVIEW

File: src/components/articles/Article.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        <ArticleImage
+          file={post.frontmatter.image.src}
+          alt={post.frontmatter.image.alt}
+        />
+      ) : null}
+      <ArticleContent>
+        <ArticleHeader
+          title={post.frontmatter.title || ''}
+          date={post.frontmatter.date || ''}
+          slug={post.fields.slug || ''}
```
Comment (zi-gae) [inline src/components/articles/Article.tsx]: How about using operator `??` to verify the falsy value?

File: src/components/articles/Article.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+const Article: React.FC<PostProps> = ({ post }) =>
+  post && post.frontmatter && post.fields ? (
+    <Container>
+      {post.frontmatter.image?.src ? (
+        <ArticleImage
+          file={post.frontmatter.image.src}
+          alt={post.frontmatter.image.alt}
+        />
+      ) : null}
```
Comment (zi-gae) [inline src/components/articles/Article.tsx]: I recommand to use operator `&&` instead of ternary operator. How about you?

**Review by zi-gae — DISMISSED** [review]
LGTM 🙇🏻‍♂️

File: src/components/articles/Article.tsx (hunk 1 above)
Comment (KennethanCeyer) [reply src/components/articles/Article.tsx]: It's a good opinion!
I'll reflect all the current old PR's code style to fit as `ts 4.4.x` version

File: src/components/articles/Article.tsx (hunk 2 above)
Comment (KennethanCeyer) [reply src/components/articles/Article.tsx]: Sound good to me
I will reflect it

Comment (KennethanCeyer) [reply src/components/articles/Article.tsx]: I'll switch it as resolved

File: src/components/articles/Article.tsx (hunk 1 above)
Comment (KennethanCeyer) [reply src/components/articles/Article.tsx]: I'll switch it as resolved

**Pull request merged**